components/Autocomplete.vue

```diff
@@ -70,10 +70,15 @@
 
 /* iteminp class is passed into this component in a prop */
 .iteminp ul {
+  /* 
```

Comment:
Whats the purpose of this? Are we trying to comment this code out? If so, remove it, if not remove the /*